libraries/ATT_IOT/CborBuilder.h

```diff
@@ -18,57 +18,58 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
+
 #ifndef CBOREN_H
```

Comment:
There's this here and KEYS_h below. Let's normalize these guards, say, CBOR_BUILDER_H & KEYS_H